patterns/news-blog-single-post-with-sidebar-template.php

```diff
@@ -14,34 +14,39 @@
 ?>
 <!-- wp:template-part {"slug":"header","area":"header"} /-->
 
-<!-- wp:group {"layout":{"type":"constrained"}} -->
-<div class="wp-block-group">
-	<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained"}} -->
+<main class="wp-block-group">
```

Comment:
I am not super happy about the amount of groups / divs in this template and it may be possible to simplify it without visual changes.